### ISSUE TITLE
does this work with forge or modded biomes?

### ISSUE THREAD

**Issue by hedkne43rei0pow** [Coun7ered/terrain-slabs-mod#2]

**Comment by Coun7ered** [comment]
No, it currently only places the slabs on blocks that I specifically added. Also it's fabric only (and will probably stay fabric only)

**Comment by hedkne43rei0pow** [comment]
It might work with sinytra connector

**Comment by hedkne43rei0pow** [comment]
also any plans to support them with like datapacks or something

**Comment by mpustovoi** [comment]
I like the idea of this mod.
I wish it worked in the presence of Tectonic, Terralith, Amplified Nether/Incendium and Nullscape at least since they don't add any blocks.

**Comment by hedkne43rei0pow** [comment]
tectonic with this mod would be amazing

**Comment by mpustovoi** [comment]
I remember that either Terralith or Tectonic do add stone slabs and snow layers to worldgen for terrain smoothness. Probably, it can be considered somehow.

**Comment by hedkne43rei0pow** [comment]
terralith does that somtimes

**Comment by souldbminerr** [comment]
Tectonic does work, and with connector, you can use my fork or wait until @Coun7ered fixes it

**Comment by souldbminerr** [comment]
> No, it currently only places the slabs on blocks that I specifically added. Also it's fabric only (and will probably stay fabric only)

Data driven slabs shouldn't be that hard if mods tag blocks to use as ground blocks (I think there is a vanilla tag which does that)
